mix.exs

```diff
@@ -28,7 +28,7 @@ defmodule Scrivener.Mixfile do
 
   defp deps do
     [
-      {:ecto, "~> 1.0"},
+      {:ecto, ">= 1.0.0", optional: true},
```

Comment:
@josevalim My intention is to use scrivener with Ecto 1 and 2 since we don't have any change in the core lib. Is it make sense?